docs/django/first-steps-with-django.rst

```diff
@@ -206,6 +206,22 @@ To use this with your project you need to follow these steps:
 
         CELERY_CACHE_BACKEND = 'django-cache'
 
+    We can also use the cache defined in the CACHES setting in django.
+
+    .. code-block:: python
+
+        # celery setting.
+        CELERY_CACHE_BACKEND = 'default'
+
+        # django setting.
+        CACHES = {
+            'default': {
+                'BACKEND': 'api.lib.redis_cache_backend.RedisCache',
```

Comment:
api??

Reply:
Just a random path.

Reply:
/path/to/cachebackend/DBbackend

Reply:
Done.

Reply:
CACHES = {
    'default': {
        'BACKEND': 'django.core.cache.backends.db.DatabaseCache',
        'LOCATION': 'my_cache_table',
    }
}

got from https://docs.djangoproject.com/en/2.2/topics/cache/

Reply:
Done.